impeller/renderer/device_capabilities.h

```diff
@@ -23,6 +23,8 @@ class IDeviceCapabilities {
 
   bool SupportsTextureToTextureBlits() const;
 
+  bool SupportsFramebufferBlending() const;
```

Comment:
`SupportsFramebufferFetch` maybe? That we use it for blending perhaps is not related to the feature being supported.

Reply:
I just saw you comments on what Apple calls it. SGTM.

Reply:
My vote if for framebuffer fetch.

Reply:
Done